projects/1.19/assets/spikyspikes/spiky-spikes/lang/zh_cn.json

```
@@ -0,0 +1,20 @@
+{
+  "block.spikyspikes.diamond_spike": "钻石尖刺",
+  "block.spikyspikes.diamond_spike.description": "造成大量伤害，生物会掉落所有战利品，就像被玩家杀死一样。可以使用大多数剑类附魔，通过铁砧赋予。",
+  "block.spikyspikes.golden_spike": "金制尖刺",
+  "block.spikyspikes.golden_spike.description": "伤害并杀死生物，只会掉落经验而没有任何战利品。",
+  "block.spikyspikes.iron_spike": "铁制尖刺",
+  "block.spikyspikes.iron_spike.description": "伤害并杀死生物，不会掉落经验，只会掉落战利品。",
+  "block.spikyspikes.netherite_spike": "下界合金尖刺",
+  "block.spikyspikes.netherite_spike.description": "造成大量伤害，生物会掉落所有战利品，就像被玩家杀死一样。可以使用大多数剑附魔，通过铁砧赋予。耐爆炸，不会被凋灵摧毁。不会对玩家造成伤害。",
+  "block.spikyspikes.stone_spike": "石制尖刺",
+  "block.spikyspikes.stone_spike.description": "伤害并杀死生物，但不会掉落任何战利品或经验。",
+  "block.spikyspikes.wooden_spike": "木制尖刺",
+  "block.spikyspikes.wooden_spike.description": "缓慢伤害生物，但不会杀死生物。",
+  "death.attack.spike": "%1$s 现在转生到了不那么尖锐的世界",
+  "death.attack.spike.player": "%1$s 监视到一个试图逃离 %2$s 的尖刺",
```

```suggestion
  "death.attack.spike.player": "%1$s在试图逃离%2$s时与尖刺来了个亲密接触",
```

oversee 在这里的意思比较迷惑。

死亡消息见 https://minecraft.fandom.com/wiki/Death_messages